scripts/create_service.sh

```diff
@@ -53,19 +58,42 @@ gcloud compute instance-templates create grpcwallet-${hostname_suffix}-template
 
 gcloud compute instance-groups managed create grpcwallet-${hostname_suffix}-mig-us-central1 \
     --zone us-central1-a \
-    --size=2 \
+    --size=${size} \
     --template=grpcwallet-${hostname_suffix}-template
 
 gcloud compute instance-groups set-named-ports grpcwallet-${hostname_suffix}-mig-us-central1 \
     --named-ports=grpcwallet-${service_type}-port:${port} \
     --zone us-central1-a 
 
-gcloud compute backend-services create grpcwallet-${hostname_suffix}-service \
-    --global \
-    --load-balancing-scheme=INTERNAL_SELF_MANAGED \
-    --protocol=GRPC \
-    --port-name=grpcwallet-${service_type}-port \
-    --health-checks grpcwallet-health-check
+project_id="$(gcloud config list --format 'value(core.project)')"
+
+backend_config="affinityCookieTtlSec: 0
+backends:
+- balancingMode: UTILIZATION
+  capacityScaler: 1.0
+  group: https://www.googleapis.com/compute/v1/projects/${project_id}/zones/us-central1-a/instanceGroups/grpcwallet-stats-mig-us-central1
+connectionDraining:
+  drainingTimeoutSec: 0
+description: ''
+healthChecks:
+- https://www.googleapis.com/compute/v1/projects/${project_id}/global/healthChecks/grpcwallet-health-check
+kind: compute#backendService
+loadBalancingScheme: INTERNAL_SELF_MANAGED
+name: grpcwallet-stats-service
+port: 80
+portName: grpcwallet-stats-port
+protocol: GRPC
+selfLink: https://www.googleapis.com/compute/v1/projects/${project_id}/global/backendServices/grpcwallet-stats-service
```

Comment:
Delete the non-required fields with default value?

And selfLink doesn't seems like an input (but an output from export)

Reply:
Done.